python/gzdice/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                gzdice
+version             0.1.2
+revision            0
+
+categories          python
+platforms           darwin
+license             MIT
+maintainers         @gleeblezoid \
+                    gleeblezoid.com:gzdice
```

Comment:
Since both the github handle and email address point to the same person, they should be enclosed in curly braces:
```
maintainers         {@gleeblezoid gleeblezoid.com:gzdice}
```

Additionally, you should add `openmaintainer` to the `maintainers` list unless there is a very specific reason why you and only you alone should maintain or make changes to this port:

```
maintainers         {@gleeblezoid gleeblezoid.com:gzdice} \
                    openmaintainer
```